src/calculateRentalCost.js

```diff
@@ -1,10 +1,20 @@
+'use strict';
+
 /**
  * @param {number} days
- *
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50; 
+  } else if (days >= 3) {
+    totalCost -= 20; 
+  }
+
+  return totalCost;
 }
```

Comment:
Consider renaming the function to something more descriptive, such as `calculateCarRentalCost` to make it clear that it's about car rental.